Userland/Libraries/LibAudio/FlacLoader.cpp

```diff
@@ -141,14 +141,39 @@ bool FlacLoaderPlugin::parse_header()
     md5_checksum.span().copy_to({ m_md5_checksum, sizeof(m_md5_checksum) });
 
     // Parse other blocks
-    // TODO: For a simple first implementation, all other blocks are skipped as allowed by the FLAC specification.
```

Comment:
Please change this to a new TODO comment that notes how we skip all block types *except* SEEKTABLE and STREAMINFO.